web-sourcecode/tests/unit/sumaClient.spec.js

```diff
@@ -194,7 +194,7 @@ describe('SumaClient.vue', () => {
     expect(initiativedata['counts']).toHaveLength(1)
     expect(wrapper.find(".countButton").text()).toBe("Count (0)")
 
-    const rightalignbuttons = wrapper.findAll('.rightalign').at(1).trigger('click')
+    // const rightalignbuttons = wrapper.findAll('.rightalign').at(1).trigger('click')
```

Comment:
Why did this get commented out? It is why the Travis CI test is breaking? Also why did this get commented out? This is submitting the counts and is a very important test.

Reply:
Ah. Okay. That's my fault. I didn't see that it was performing an action.  The linter complained that the variable is never used. We should be able to just remove the variable declaration part of that.

Reply:
I just committed a fix. Thanks for the catch @dnoneill!